spec/rubocop/cop/style/inverse_methods_spec.rb

```diff
@@ -150,6 +150,36 @@
     end
   end
 
+  it 'allows comparing camel case constants on the right' do
+    expect_no_offenses(<<-RUBY.strip_indent)
+      klass = self.class
+      !(klass < BaseClass)
+    RUBY
+  end
+
+  it 'allows comparing camel case constants on the left' do
+    expect_no_offenses(<<-RUBY.strip_indent)
+      klass = self.class
+      !(BaseClass < klass)
+    RUBY
+  end
+
+  it 'allows comparing snake case constants on the right' do
+    expect_offense(<<-RUBY.strip_indent)
+      klass = self.class
+      !(klass < FOO_BAR)
```

Comment:
Btw, why allow `SNAKE_CASE`?

Reply:
Whoops, copy/paste issue. The name of the test is wrong. This is expected to be an offense.